g2p/cli.py

```diff
@@ -172,13 +178,34 @@ def convert(
     # Check valid input
     # Check input != output
     if in_lang == out_lang:
-        raise click.UsageError("Values must be different for 'IN_LANG' and 'OUT_LANG'")
+        raise click.UsageError(
+            "Values must be different for 'IN_LANG' and 'OUT_LANG'")
+    if config:
+        # This isn't that DRY - copied from g2p/mappings/langs/__init__.py
+        mappings_legal_pairs = []
+        with open(config, encoding='utf8') as f:
+            data = yaml.safe_load(f)
+        if 'mappings' in data:
+            for index, mapping in enumerate(data['mappings']):
+                mappings_legal_pairs.append(
+                    (data['mappings'][index]['in_lang'], data['mappings'][index]['out_lang']))
+                data['mappings'][index] = load_mapping_from_path(config, index)
+        else:
+            data = load_mapping_from_path(config)
+        for pair in mappings_legal_pairs:
+            if pair[0] in LANGS_NETWORK.nodes:
+                LOGGER.warn(
+                    f"A mapping with the name '{pair[0]}' is already defined in g2p. Your local mapping with the same name might not function properly.")
+        LANGS_NETWORK.add_edges_from(mappings_legal_pairs)
+        MAPPINGS_AVAILABLE.extend(data['mappings'])
```

Comment:
So, if I understand this code correctly, `--config` adds to the existing network, rather than wholesale replacing the existing network. That seems much more convenient, so I agree with the idea, I had just understood it the other way around, that only what's in the given config would be usage.